user-edit.css

```diff
@@ -34,4 +34,24 @@
 
 .two-factor-methods-table tbody tr:nth-child(odd) {
 	background-color: #f9f9f9;
-}
\ No newline at end of file
+}
+
+@media print {
+	body * {
+		visibility: hidden;
+		max-height: 100px;
```

Comment:
display doesn't work because it hides all children, and you can't use display to get children back if the parent isn't shown.